### PR TITLE
Bug 1970129: Add env variable OVS_SYS_LOG_LEVEL for ovn nodes to setup ovs syslog level

### PR DESCRIPTION
**Pull request by npinaeva**
Since OVS is run on host, OVS logs are collected from systemd journal. Default value for ovs syslog level is err, set it to info to collect info level logs with must-gather.

How to test:
Run `ovs-appctl vlog/list` on a node, assure syslog level is set to requested level.
Run must-gather, check `host_service_logs/<node_name>/ovs-vswitchs_service.log` file. Previously it only contained ERR messages like this:

> Jun 28 07:35:19.248737 ci-ln-hxbr65t-f76d1-zw5kl-master-0 ovs-vswitchd[1234]: ovs|00040|stream_ssl|ERR|SSL_use_PrivateKey_file: error:20074002:BIO routines:file_ctrl:system lib
Jun 28 07:35:19.248780 ci-ln-hxbr65t-f76d1-zw5kl-master-0 ovs-vswitchd[1234]: ovs|00041|stream_ssl|ERR|failed to load client certificates from /ovn-ca/ca-bundle.crt: error:140AD002:SSL routines:SSL_CTX_use_certificate_file:system lib

After setting default level to INFO it should have INFO messages like this:
> Jun 28 08:54:14.080482 ci-ln-hxbr65t-f76d1-zw5kl-master-0 ovs-vswitchd[1234]: ovs|00507|connmgr|INFO|br-ex<->unix#1028: 2 flow_mods in the last 0 s (2 adds)
Jun 28 08:54:21.020662 ci-ln-hxbr65t-f76d1-zw5kl-master-0 ovs-vswitchd[1234]: ovs|00508|connmgr|INFO|br-int<->unix#1020: 5123 flow_mods 10 s ago (5122 adds, 1 deletes)

You can change default OVS_LOG_LEVEL value with ConfigMap:
```yaml
kind: ConfigMap
apiVersion: v1
metadata:
  name: env-overrides
  namespace: openshift-ovn-kubernetes
  annotations:
data:
  # to set the node processes on a single node to verbose
  # replace this with the node's name (from oc get nodes)
  ip-10-0-135-96.us-east-2.compute.internal: |
    OVN_KUBE_LOG_LEVEL=5
    OVN_LOG_LEVEL=dbg
    OVS_LOG_LEVEL=dbg
  # to adjust master log levels, use _master
  _master: |
    OVN_KUBE_LOG_LEVEL=5
    OVN_LOG_LEVEL=dbg
```

### CODE REVIEW

**Comment by tssurya** [comment]
/lgtm

Simple fix to change the ovs log level.
Based on the comment here: https://github.com/openshift/must-gather/pull/243#discussion_r657090694 from @trozet seems like the right way to go.

**Comment by tssurya** [comment]
CI failures seem unrelated

File: README.md [hunk 1]
```diff
@@ -229,6 +229,7 @@ data:
   ip-10-0-135-96.us-east-2.compute.internal: |
     OVN_KUBE_LOG_LEVEL=5
     OVN_LOG_LEVEL=dbg
+    OVS_LOG_LEVEL=dbg
```
Comment (abhat) [inline README.md:232]: should we call this `OVS_SYS_LOG_LEVEL`? That makes it more explicit, since you could set console and file logging too.

Comment (npinaeva) [reply README.md:232]: Sure, explicit is always better

**Pull request merged**